Marlin/language_en.h

```diff
@@ -508,5 +508,8 @@
 #ifndef MSG_DELTA_CALIBRATE_CENTER
   #define MSG_DELTA_CALIBRATE_CENTER          "Calibrate Center"
 #endif
-
+#define MSG_CHANGE_FILAMENT                 "Filament"
+#define MSG_REMOVE_FILAMENT                 "Remove"
+#define MSG_INSERT_FILAMENT                 "Insert"
+#define MSG_INSERT_FILAMENT_SLOW            "Slow Insert"
 #endif // LANGUAGE_EN_H
```

Comment:
Yes " rhetorical questions" and "sarcasm" have been addressed to me.

Ehhh, 'variables'?
No warnings?
```
                 from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\temperature.cpp:46:

C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language_en.h:511:0: warning: "MSG_CHANGE_FILAMENT" redefined [enabled by default]

 #define MSG_CHANGE_FILAMENT                 "Filament"

 ^

In file included from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language.h:250:0,

                 from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\temperature.cpp:46:

C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language_pt-br.h:174:0: note: this is the location of the previous definition

 #define MSG_CHANGE_FILAMENT                 "Filamento"

 ^

In file included from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language.h:251:0,

                 from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\temperature.cpp:46:

C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language_en.h:512:0: warning: "MSG_REMOVE_FILAMENT" redefined [enabled by default]

 #define MSG_REMOVE_FILAMENT                 "Remove"

 ^

In file included from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language.h:250:0,

                 from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\temperature.cpp:46:

C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language_pt-br.h:175:0: note: this is the location of the previous definition

 #define MSG_REMOVE_FILAMENT                 "Retirar"

 ^

In file included from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language.h:251:0,

                 from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\temperature.cpp:46:

C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language_en.h:513:0: warning: "MSG_INSERT_FILAMENT" redefined [enabled by default]

 #define MSG_INSERT_FILAMENT                 "Insert"

 ^

In file included from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language.h:250:0,

                 from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\temperature.cpp:46:

C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language_pt-br.h:176:0: note: this is the location of the previous definition

 #define MSG_INSERT_FILAMENT                 "Inserir"

 ^

In file included from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language.h:251:0,

                 from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\temperature.cpp:46:

C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language_en.h:514:0: warning: "MSG_INSERT_FILAMENT_SLOW" redefined [enabled by default]

 #define MSG_INSERT_FILAMENT_SLOW            "Slow Insert"

 ^

In file included from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language.h:250:0,

                 from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\temperature.cpp:46:

C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language_pt-br.h:177:0: note: this is the location of the previous definition

 #define MSG_INSERT_FILAMENT_SLOW            "Lentamente"

 ^

"C:\Program Files (x86)\Arduino\hardware\tools\avr/bin/avr-g++" -c -g -Os -Wall -Wextra -std=gnu++11 -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics -MMD -mmcu=atmega2560 -DF_CPU=16000000L -DARDUINO=10608 -DARDUINO_AVR_MEGA2560 -DARDUINO_ARCH_AVR -fverbose-asm -save-temps  "-IC:\Users\ah\AppData\Local\Arduino15\packages\arduino\hardware\avr\1.6.10\cores\arduino" "-IC:\Users\ah\AppData\Local\Arduino15\packages\arduino\hardware\avr\1.6.10\variants\mega" "-IC:\Users\ah\Documents\Arduino\libraries\U8glib\src" "C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\twibus.cpp" -o "C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\twibus.cpp.o"
"C:\Program Files (x86)\Arduino\hardware\tools\avr/bin/avr-g++" -c -g -Os -Wall -Wextra -std=gnu++11 -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics -MMD -mmcu=atmega2560 -DF_CPU=16000000L -DARDUINO=10608 -DARDUINO_AVR_MEGA2560 -DARDUINO_ARCH_AVR -fverbose-asm -save-temps  "-IC:\Users\ah\AppData\Local\Arduino15\packages\arduino\hardware\avr\1.6.10\cores\arduino" "-IC:\Users\ah\AppData\Local\Arduino15\packages\arduino\hardware\avr\1.6.10\variants\mega" "-IC:\Users\ah\Documents\Arduino\libraries\U8glib\src" "C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\ultralcd.cpp" -o "C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\ultralcd.cpp.o"
In file included from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language.h:251:0,

                 from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\ultralcd.cpp:26:

C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language_en.h:511:0: warning: "MSG_CHANGE_FILAMENT" redefined [enabled by default]

 #define MSG_CHANGE_FILAMENT                 "Filament"

 ^

In file included from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language.h:250:0,

                 from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\ultralcd.cpp:26:

C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language_pt-br.h:174:0: note: this is the location of the previous definition

 #define MSG_CHANGE_FILAMENT                 "Filamento"

 ^

In file included from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language.h:251:0,

                 from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\ultralcd.cpp:26:

C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language_en.h:512:0: warning: "MSG_REMOVE_FILAMENT" redefined [enabled by default]

 #define MSG_REMOVE_FILAMENT                 "Remove"

 ^

In file included from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language.h:250:0,

                 from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\ultralcd.cpp:26:

C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language_pt-br.h:175:0: note: this is the location of the previous definition

 #define MSG_REMOVE_FILAMENT                 "Retirar"

 ^

In file included from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language.h:251:0,

                 from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\ultralcd.cpp:26:

C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language_en.h:513:0: warning: "MSG_INSERT_FILAMENT" redefined [enabled by default]

 #define MSG_INSERT_FILAMENT                 "Insert"

 ^

In file included from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language.h:250:0,

                 from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\ultralcd.cpp:26:

C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language_pt-br.h:176:0: note: this is the location of the previous definition

 #define MSG_INSERT_FILAMENT                 "Inserir"

 ^

In file included from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language.h:251:0,

                 from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\ultralcd.cpp:26:

C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language_en.h:514:0: warning: "MSG_INSERT_FILAMENT_SLOW" redefined [enabled by default]

 #define MSG_INSERT_FILAMENT_SLOW            "Slow Insert"

 ^

In file included from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language.h:250:0,

                 from C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\ultralcd.cpp:26:

C:\Users\ah\AppData\Local\Temp\build0ca64bc2097fb2df5a6ea132866e93fd.tmp\sketch\language_pt-br.h:177:0: note: this is the location of the previous definition

 #define MSG_INSERT_FILAMENT_SLOW            "Lentamente"

 ^
```